src/main/java/uk/gov/pay/products/validations/PanDetector.java

```diff
@@ -10,10 +10,12 @@ public static boolean isSuspectedPan(String reference) {
         if (reference.length() < MIN_PAN_LENGTH || reference.length() > MAX_PAN_LENGTH) {
             return false;
         }
-        var cleaned = reference
-                .replace("-", "")
-                .replace(" ", "");
+        var cleaned = cleanedReference(reference);
```

Comment:
Maybe this `var` should be `String` now as it’s not 100% clear from the method name what it returns.